pkg/reconcilermanager/controllers/reconciler_base.go

```diff
@@ -501,6 +503,22 @@ func mutateContainerResource(c *corev1.Container, override *v1beta1.OverrideSpec
 			}
 		}
 	}
+	var found = false
```

Comment:
This whole added code chunk could probably be extracted to a `containerLogLevel(containerName string)` function and then just always append the -v flag with the value, even if it's zero.

Reply:
done. created `containerLogLevel(containerName string, override []v1beta1.ContainerLogLevelOverride) int` that will determine log level value and simplified the logic